src/module/rules/language.mjs

```
@@ -1,39 +1,41 @@
+export function language() {
+    console.log("Elkan 5e  |  Initializing Languages");
+    
+    const LANGUAGES_TO_DELETE = ["giant", "gnomish", "orc", "druidic", "cant"];
+    
+    if (!CONFIG.DND5E?.languages?.standard?.children) {
+        console.warn("Elkan 5e | CONFIG.DND5E.languages not properly initialized");
+        return
+    }
+    // Deleting Languages
+    LANGUAGES_TO_DELETE.forEach(lang => delete CONFIG.DND5E.languages.standard.children[lang]);
+    delete CONFIG.DND5E.languages.exotic;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)